tests/client/conftest.py

```diff
@@ -69,7 +69,7 @@ def is_using_real_services(request):
 
 @pytest.fixture(scope='session')
 def client():
-    return ScrapinghubClient(auth=TEST_ADMIN_AUTH,
+    return ScrapinghubClient(apikey=TEST_ADMIN_AUTH,
```

Comment:
Sorry for not being sound enough about the change, that's my bad.

Right, I think it will never happen, my args are the following:
- `Connection` works only with [apikey](https://github.com/scrapinghub/python-scrapinghub/blob/master/scrapinghub/legacy.py#L83), `HubstorageClient` [logic](https://github.com/scrapinghub/python-scrapinghub/blob/master/scrapinghub/legacy.py#L83) is more flexible, it also accepts `(user,pass)` as a tuple, or as a colon-separated auth string, so apikey looks like a common denominator for both clients, simple and power enough;
- apikey key looks like an enhancement over user/pass pair, and as last approach was dropped from `Connection` at some [point](https://github.com/scrapinghub/python-scrapinghub/commit/6655c2b67e8e21a95a5d85fd9694f1bc1b8cc40f#diff-5908bb1e4376e7f4b8bcf236a31c9487R52), I think we're not going to revise it in the nearest future.

Anyway let's revise it if you have something in mind.